src/parsers/RssParser.js

```diff
@@ -170,9 +170,11 @@ class RssParser {
       let inputKey = attribute
       let resultKey = attribute
 
-      if (attribute instanceof Object) {
+      if (attribute instanceof Object && !data[attribute.as]) {
```

Comment:
tu kontrolu jestli atribut existuje bych nemixoval s logikou, která se rozhoduje, co se bude hledat, radši bych to přesunul k přiřazení hodnoty do výsledku